scripts/partition_manager/partition_manager.rst

```diff
@@ -15,11 +15,16 @@ The Partition Manager is activated for all multi-image builds, no matter what bu
 Overview
 ********
 
-The Partition Manager script reads configuration files named :file:`pm.yml`, which define flash partitions.
-A partition's definition includes its name and constraints on its size and placement on flash.
+The Partition Manager script reads configuration files named :file:`pm.yml`, which define flash and RAM partitions.
+A flash partition's definition includes its name and constraints on its size and placement on flash.
+A RAM partition's definition includes its name and constraints on its size.
 The Partition Manager allocates a start address and sometimes a size to each partition in a way that satisfies these constraints.
 
-There are different kinds of partitions:
+There are four different kinds of **flash partitions**, and three different kinds of
+**ram partitions**, as described below.
```

Comment:
```suggestion
There are different kinds of **flash partitions** and **RAM partitions**, as described below.
```

You will forget to update this sentence every time you add a new type, so let's be less specific. ;)